test/lib/ansible_test/_data/sanity/code-smell/runtime.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
```

Comment:
Should we perhaps call this test `runtime-metadata` instead of just `runtime`?

I think `runtime` by itself is rather ambiguous.